# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/kasc/xblasttnt/linux-joystick_h.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./x11_joystick.c.orig	2005-11-04 17:50:18.000000000 +0100
+++ ./x11_joystick.c	2005-11-04 17:50:43.000000000 +0100
@@ -32,7 +32,18 @@
 #ifndef NO_JOYSTICK
 #include <fcntl.h>
 #include <unistd.h>
+
+#ifdef __STRICT_ANSI__
+#define REDEFINE__STRICT_ANSI__
+#undef __STRICT_ANSI__
+#endif
+
 #include <linux/joystick.h>
+
+#ifdef REDEFINE__STRICT_ANSI__
+#define __STRICT_ANSI__
+#endif
+
 #endif
 
 #define JOYDIR_NONE  0x00
